formatter/src/source_file.rs

```diff
@@ -136,7 +136,7 @@ mod tests {
     }
 
     #[test]
-    fn override_html_macro() {
+    fn override_format_macros() {
```

Comment:
can you add one more test with a fully qualified path (for example `foo::bar::some_view!`)?

Reply:
I have added two of these, one which includes multiple indented macros